internal/onprem/event_extractor.go

```diff
@@ -183,13 +183,41 @@ func (e *EventExtractor) extractEvents(tmpdir string, eventChannel chan types.Ev
 		},
 	}
 
+	versions, err := getVersionsFromFile(tmpdir, "/*/versions.json")
+	if err != nil {
+		e.logger.WithFields(logrus.Fields{
+			"err": err,
+		}).Warning("error extracting versions")
```

Comment:
isn't it worth to log the cluster id as well?

Reply:
We don't yet have it though :sweat_smile: 
We have the tar filename which might help put together the errors, we could propagate that through